.circleci/config.yml

```diff
@@ -6,38 +6,32 @@ references:
         only: master
 
 jobs:
-  publish:
+  build:
     macos:
       xcode: "15.0.0"
+    resource_class: macos.x86.medium.gen2
```

Comment:
This will break in a month

https://discuss.circleci.com/t/macos-intel-support-deprecation-in-january-2024/48718

Use apple silicon

Reply:
i know

Reply:
Ok, I will let a current employee approve then :)